sources/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -70,10 +70,7 @@ save_journal() {
     # Q: would it be possible that journal will not be flushed completely yet?
     echo "writing logs to disk and rebooting"
 
-    local logfile="$NEWROOT/var/log/upgrade.log"
-
-    # Add a separator if file exists
-    [ -e $logfile ] && echo "### LEAPP reboot ###" >> $logfile
```

Comment:
Is there any specific reason to remove this separator?

Reply:
I removed because logs are archived but I added again because of boot loop.